ql/src/test/queries/clientpositive/cbo_join_transitive_pred_loop_1.q

```diff
@@ -1,5 +1,8 @@
 set hive.optimize.join.disjunctive.transitive.predicates.pushdown=false;
 
+-- HiveProject(month=[CAST(202503):INTEGER])
+--! qt:replace:/(.*month=\[CAST\()\d+(\).*)/$1#Masked#$2/
```

Comment:
Instead of regex replace I think it will be safer and more robust to new plan changes to use `hive.test.currenttimestamp`.

Reply:
IMHO the presence of setting like `hive.test.currenttimestamp` is not safe. Test code should not be shipped with production code.

Reply:
I am not a big fan of such settings either but since we have them we could as well use them. Anyways, don't feel strongly about it so pick whatever option you feel its best.